<launch>
	<node name="speed_controller" pkg="iarrc" type="speed_controller" output="screen">
        <param name="go_speed" value="12"/>
        <param name="max_crosses" value="1"/>
	</node>
</launch>
